Woof/WoofTests/Model/Preferences/PreferencesHandlerTestHelper.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/**
+  A helper struct for handling preferences in tests.
+ - Note: This struct is intended for use in test cases only.
+ */
+struct PreferencesHandlerTestHelper {
+    /**
+     Cleans the storage used for preferences.
+     This method deletes the user preferences data from the key-value storage.
+
+     Example usage:
+     ```
+     PreferencesHandlerTestHelper.cleanStorage()
+     ```
+     */
+    static func cleanStorage() {
```

Comment:
The name is confusing.
You do not clean the storage. You just removes one key from the one specific storage.

Reply:
I reworked the code and removed this function, instead I use the method to delete from KeyValueStorage